ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.js

```diff
@@ -40,6 +41,7 @@ const SnapUIRendererComponent = ({
   useFooter = false,
   onCancel,
   contentBackgroundColor,
+  disableCustomBackground = false,
```

Comment:
Instead of introducing this, I think we should just force the color if `contentBackgroundColor` is defined. Then we can pass in the explicit color we want where it needs to be forced to a certain color.

To do that, we would need to swap the ordering in the definition of `backgroundColor` below.